index.d.ts

```diff
@@ -0,0 +1,32 @@
+/** Declaration file generated by dts-gen */
+
+export function createClass(...args: any[]): Class<GeneratedClass>;
+
+export function createContainer(...args: any[]): Class<Container>;
+
+export function resolveContainer(Container: any): Container;
```

Comment:
In the above 3 functions we don't know anything about the parameters? So can't we be more specific than `any`?